src/Extensions/Features/src/FeatureCollection.cs

```diff
@@ -140,6 +140,9 @@ public void Set<TFeature>(TFeature? instance)
         this[typeof(TFeature)] = instance;
     }
 
+    // Used by the debugger. Count over enumerable is required to get the correct value.
+    internal int GetCount() => this.Count();
```

Comment:
```suggestion
    private int GetCount() => this.Count();
```
To hide the debugger-detail a bit more?